### PR TITLE
Add documentation on zwave_js automations

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
We are adding a new zwave_js automation trigger in https://github.com/home-assistant/core/pull/54897 so adding some documentation on automations.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54897
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -413,6 +413,62 @@ action:
...
+  property_key: null
+  endpoint: 0
+  # `from` and `to` will both accept lists of values and the trigger will fire if the value update matches any of the listed values
+  from:
+    - closed
+    - jammed
+  to: opened
+```
+
+{% endraw %}
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]: - We do not need to wrap this in raw, as there is no templating involved.
- Added some quotes to follow our YAML style guide lines

````suggestion
```yaml
# Example automation trigger that fires whenever the `latchStatus` value changes from `closed` to `opened` on the three devices (devices will be derived from an entity ID).
trigger:
  platform: zwave_js.value_updated
  # At least one `device_id` or `entity_id` must be provided
  device_id: 45d7d3230dbb7441473ec883dab294d4  # Garage Door Lock device ID
  entity_id:
    - lock.front_lock
    - lock.back_door
  # `property` and `command_class` are required
  command_class: 98 # Door Lock CC
  property: "latchStatus"
  # `property_key` and `endpoint` are optional
  property_key: null
  endpoint: 0
  # `from` and `to` will both accept lists of values and the trigger will fire if the value update matches any of the listed values
  from:
    - "closed"
    - "jammed"
  to: "opened"
```
````

**Review by frenck — COMMENTED** [review]
Small suggestion/request, otherwise it looks good 👍

**Review by frenck — CHANGES_REQUESTED** [review]
Correct review state...

File: source/_integrations/zwave_js.markdown (hunk 1 above)
Comment (raman325) [reply source/_integrations/zwave_js.markdown]: thanks @frenck ! I will have to review the style guide, forgot about those 😅

**Review by frenck — APPROVED** [review]
Thanks, @raman325 👍

**Pull request merged**